scripts/session.js

```diff
@@ -5,6 +5,11 @@ var sessionName = Cookies.get("session_name");
 var userName = Cookies.get("user_name");
 var cookie_point = Cookies.get("point");
 
+// redirect to login page
+if(sessionName == undefined) {
```

Comment:
`if (!sessionName)` is enough.

In addition, I think we should check both user name and session name here.